apps/builder/app/builder/features/sidebar-left/panels/pages/page-settings.tsx

```diff
@@ -755,6 +775,29 @@ const FormFields = ({
                   </Flex>
                 </PanelBanner>
               )}
+
+              {isFeatureEnabled("xmlElement") && (
+                <Grid gap={1}>
+                  <Label htmlFor={fieldIds.documentType}>Document Type</Label>
+                  <Select
+                    options={documentTypes}
+                    getValue={(docType: (typeof documentTypes)[number]) =>
```

Comment:
I think you can pass `<Select<DocumentType>` to avoid all these annotations

Reply:
No, Select types are broken